docs/source/notes/autograd.rst

```diff
@@ -368,9 +368,9 @@ Thread Safety on Autograd Node
 
 Since Autograd allows the caller thread to drive its backward execution for
 potential parallelism, it's important that we ensure thread safety on CPU with
-parallel backwards that share part/whole of the GraphTask.
+parallel ``backward()`` calls that share part/whole of the GraphTask.
 
-Custom Python ``autograd.Function`` is automatically thread safe because of GIL.
+Custom Python ``autograd.Function``s are automatically thread safe because of GIL.
```

Comment:
This change broke the rendering, compare https://github.com/pytorch/pytorch/blob/a9b17decd7af4fbe1295b112d9c2d255fb1e0fdd/docs/source/notes/autograd.rst#thread-safety-on-autograd-node and https://github.com/pytorch/pytorch/blob/master/docs/source/notes/autograd.rst#thread-safety-on-autograd-node

Reply:
Good catch, sorry about that. It's now fixed